### DISCUSSION TITLE
Masque Classic

### DISCUSSION THREAD

**Discussion by Horodej** [SFX-WoW/Masque#206] [Issues] [answered]
Hi, after new patch Masque is broken - after login in game, stance bar is broken. Vissible only Battle stance. After disable Masque, problem is gone. Can u fix it ??? Thanks !!! :)

**Comment by StormFX** [comment]
This is an issue with the latest version(s) of ElvUI.

**Comment by Horodej** [comment]
No, ElvUI update to last version 1.19 and problem still exist.

BROKEN BUFFS / DEBUFFS / ACTIONBARS:

# Disable Masque and wait for them to push an update.

**Comment by StormFX** [comment]
As I stated above, This is an ElvUI problem. To verify, type the following in the chat window:

`/luaerror on`

And report any errors you get.

**Comment by StormFX** [comment, marked as answer]
This issue is related to https://www.tukui.org/forum/viewtopic.php?f=40&t=9602. Closing.
